lib/widgets/message_list.dart

```diff
@@ -1262,15 +1270,24 @@ class MessageWithPossibleSender extends StatelessWidget {
           if (senderRow != null)
             Padding(padding: const EdgeInsets.fromLTRB(16, 2, 16, 0),
               child: senderRow),
-          EditStateMarker(
-            editState: message.editState,
+          Row(
+            crossAxisAlignment: CrossAxisAlignment.baseline,
+            textBaseline: localizedTextBaseline(context),
             children: [
+              const SizedBox(width: 16),
               Expanded(child: Column(
                 crossAxisAlignment: CrossAxisAlignment.stretch,
                 children: [
                   MessageContent(message: message, content: item.content),
                   if ((message.reactions?.total ?? 0) > 0)
-                    ReactionChipsList(messageId: message.id, reactions: message.reactions!)
+                    ReactionChipsList(messageId: message.id, reactions: message.reactions!),
+                  if (editStateText != null)
+                    Text(editStateText, textAlign: TextAlign.end,
+                      style: TextStyle(
+                        color: messageListTheme.editStateLabel,
+                        fontFamily: 'Source Sans 3',
+                        fontSize: 12,
+                        height: (12 / 12))),
```

Comment:
```suggestion
                        height: (12 / 12),
                        letterSpacing: proportionalLetterSpacing(context, 0.05, baseFontSize: 12),
```

I notice the Figma has 0.6px letter spacing, but if we use `proportionalLetterSpacing`, it'll grow and shrink with the system font-size setting, and that seems like a nice refinement.

Reply:
Oh interesting, so yeah, it looks like the 5% is expressed in the Figma too :) thanks for noticing that.